docs/src/examples/components/Slider/Variations/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Variations = () => (
+  <ExampleSection title="Variations">
+    <ComponentExample
+      title="Icon"
+      description="A slider can have an icon."
+      examplePath="components/Slider/Variations/SliderExampleIcon"
```

Comment:
`icon` and `iconPosition` can be combined into single example, what do you think?

And also placed under `Slots` section

Reply:
done thx